ros/rolling/src/msft_ros2_env/package.xml

```diff
@@ -0,0 +1,15 @@
+<package format="2">
+  <name>msft_ros2_env</name>
+  <version>0.0.1</version>
+  <description>The package provides the environment variables useful for ROS on Windows.</description>
+  <maintainer email="seanyen@microsoft.com">Sean Yen</maintainer>
```

Comment:
Can you make this riotsquad@microsoft.com?

Reply:
updated.